Kitodo/src/main/resources/kitodo_config.properties

```diff
@@ -142,24 +127,6 @@ localMessages=/usr/local/kitodo/messages/
 # language.force-default=de
 # Default language: If no Accept-Language Http Request header is present, use the following language:
 language.default=de
-# Resource bundle key for the text to show on the primary screen.
-# - allgemeinesTextDemo: predefined with a text listing the example logins
-#       available after setup
-# - allgemeinesText: predefined with a text about the application
-ApplicationHomepageMsg=allgemeinesTextDemo
```

Comment:
`allgemeinesTextDemo` and `allgemeinesText` are used inside translation files. If they get removed here, they could be removed in translation files too.